templates/csharp/ai-bot/GettingStarted.md

```diff
@@ -0,0 +1,93 @@
+# Welcome to Teams Toolkit!
+
+This template showcases a bot app that responds to user questions like an AI assistant. This enables your users to talk with the AI assistant in Teams to find information.
+
+The app template is built using the Teams AI library, which provides the capabilities to build AI-based Teams applications.
+
+## Quick Start
+
+**Prerequisites**
+> To run the AI Chat Bot template in your local dev machine, you will need:
+>
+> - [.NET 6.0 SDK](https://dotnet.microsoft.com/download/dotnet/6.0)
+> - A [Microsoft 365 account for development](https://docs.microsoft.com/microsoftteams/platform/toolkit/accounts)
+> - [Azure OpenAI](https://aka.ms/oai/access) resource or an account with [OpenAI](https://platform.openai.com).
+
+1. Set your [OpenAI API Key](https://openai.com/api/) to *appsettings.Development.json*.
+
+    ```json
+      "OpenAI": {
+        "ApiKey": "<your-openai-api-key>"
+      },
+    ```
+1. In the debug dropdown menu, select Dev Tunnels > Create A Tunnel (set authentication type to Public) or select an existing public dev tunnel
```

Comment:
recommend to make it:
.... > Create A Tunnel, set authentication type to Public, or .....

Reply:
Done.